contracts/sumtree-orderbook/src/order.rs

```diff
@@ -669,9 +667,6 @@ pub(crate) fn claim_order(
     // Immutable amount to prevent bounty/maker fee calculations affecting each other
     let raw_amount = amount;
 
-    // Cannot send a zero amount, may be zero'd out by rounding
-    ensure!(!amount.is_zero(), ContractError::ZeroClaim);
-
```

Comment:
Since we are skipping this check now, we should also probably be skipping the bounty & maker fee calculations if the amount is zero right?